src/components/CommandPalette.tsx

```diff
@@ -231,7 +243,15 @@ const CommandRow: FC<{
             className={css({
               minWidth: '4em',
               whiteSpace: 'nowrap',
-              color: disabled ? 'gray' : gestureInProgress === shortcut.gesture ? 'vividHighlight' : 'fg',
+              color: disabled
+                ? 'gray'
+                : isTouch
+                  ? selected
+                    ? 'vividHighlight'
+                    : 'fg'
+                  : gestureInProgress === shortcut.gesture
+                    ? 'vividHighlight'
+                    : 'fg',
```

Comment:
You can eliminate the duplication here by converting the two separate inner conditions into a single union:

```ts
    : isTouch
      ? selected || gestureInProgress === shortcut.gesture
        ? 'vividHighlight'
        : 'fg',
```